public/contents/3-build-your-first-web-app/tours.json

```diff
@@ -58,7 +58,7 @@
       "fileSize": "29.58 KB",
```

Comment:
@manoj-surisetty Checksums for 4 other zips that have been modified are missing.